File: Examples/JXPagerViewExample-OC/JXPagerViewExample-OC/Example/Zoom/ZoomViewController.m
//
//  ZoomViewController.m
//  JXPagerViewExample-OC
//
//  Created by jiaxin on 2018/9/12.
//  Copyright © 2018年 jiaxin. All rights reserved.
//

#import "ZoomViewController.h"

@implementation ZoomViewController

- (void)mainTableViewDidScroll:(UIScrollView *)scrollView {
    [self.userHeaderView scrollViewDidScroll:scrollView.contentOffset.y];
}

@end

File: Examples/JXPagingViewExample/JXPagingViewExample/Example/Zoom/ZoomViewController.swift
//
//  ZoomViewController.swift
//  JXPagingView
//
//  Created by jiaxin on 2018/8/8.
//  Copyright © 2018年 jiaxin. All rights reserved.
//

import UIKit
import JXPagingView

class ZoomViewController: BaseViewController {

    override func viewDidLoad() {
        super.viewDidLoad()

    }

    override func pagingView(_ pagingView: JXPagingView, mainTableViewDidScroll scrollView: UIScrollView) {
        userHeaderView.scrollViewDidScroll(contentOffsetY: scrollView.contentOffset.y)
    }
}
